internal/node/node_launcher.sh

```diff
@@ -129,8 +129,19 @@ for ARG in "${ALL_ARGS[@]}"; do
   esac
 done
 
+# Note: Bash does not forward any termination signals to any child process when running in docker
+# so need to manually trap and forward the signals
+_term() {
+  kill -TERM "$child" 2>/dev/null
```

Comment:
Does this mean that if docker *isn't* being used, the process ends up receiving two signals? (once as part of the foreground process group, and once from this code)

Another thought: If the subprocess spawns even more subprocesses, does this work correctly?

Reply:
Just tested it:

```
process.on('SIGINT', function (code) {
  console.log('SIGINT received...');
  setTimeout(() => process.exit(0), 5000);
});
```

and the process just prints the log once.

I also tested spawning up a child process and the behaviour is the same before and after the change, the child process does not get the termination signal forwarded.